File: Serper.Api/Entity/Knowled/KnowledGegraph.cs
namespace Serper.API.Entity.Knowled
{
    public sealed class KnowledGegraph
    {
        public string title { get; set; }

        public string type { get; set; }

        public string website { get; set; }

        public string imageUrl { get; set; }

        public string description { get; set; }

        public string descriptionSource { get; set; }

        public string descriptionLink { get; set; }

        public Attributes attributes { get; set; }
    }
}

File: Serper.Web/ClientApp/src/app/entity/KnowledGegraph.ts
import { Attributes } from "./Attributes";

export class KnowledGegraph {
  constructor(
    public title?: string,

    public type?: string,

    public website?: string,

    public imageUrl?: string,

    public description?: string,

    public descriptionSource?: string,

    public descriptionLink?: string,

    public attributes?: Attributes

  ) { }
}
